cmd/utils.go

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
this utility is added to read and parser CLI `--param` and `--param-file`. All key value pairs from `--param` are stored in `utils.Flags.param`, also error is reported if `--param` does not have key value pair. Same for `--param-file`, error is reported if it does not have a file name or invalid file name, else, the entire JSON payload is read and stored in `utils.Flags.param` overwriting the ones specified in `--param`.